src/components/Cart/CartItemView/CartItem.jsx

```diff
@@ -0,0 +1,67 @@
+import { updateCartItem, deleteCartItem } from '../../../services/index';
+import { useFilter } from '../../../contexts/index';
+import { useAuth } from '../../../contexts/index';
+
+const CartItem = () => {
+  const {state:{cart}, dispatch} = useFilter();
+  const {state: {payload: {token}}} = useAuth(); 
+
+  const updateCartItemHandler = (id, data) => {
+    updateCartItem(dispatch, id, data, token);
```

Comment:
instead of passing down a token like this, give the token to the API call from the function itself , will save few line code